packages/lib-classifier/src/components/Classifier/components/Banners/components/SubjectSetProgressBanner/SubjectSetProgressBanner.js

```diff
@@ -20,29 +17,27 @@ function SubjectSetProgressBanner({
   subject,
   workflow
 }) {
+  const { t } = useTranslation('components')
+
   const [ showModal, setShowModal ] = useState(false)
   const [ intent, setIntent ] = useState(undefined)
 
   const setName = workflow?.subjectSet?.display_name || ''
   const subjectTotal = workflow?.subjectSet?.set_member_subjects_count
   const background = (subject?.alreadySeen || subject?.retired) ? 'status-critical' : 'status-ok'
   const color = (subject?.alreadySeen || subject?.retired) ? 'neutral-6' : 'neutral-7'
-  let statusText = subject?.alreadySeen ? `${counterpart('SubjectSetProgressBanner.alreadySeen')}` : ''
-  statusText = subject?.retired ? `${counterpart('SubjectSetProgressBanner.finished')}` : statusText
-  const progressText = counterpart('SubjectSetProgressBanner.bannerText', {
-    number: subject?.priority,
-    setName,
-    total: subjectTotal
-  })
-  const tooltipText = counterpart('SubjectSetProgressBanner.tooltipText')
+  let statusText = subject?.alreadySeen ? `${t('Banners.SubjectSetProgressBanner.alreadySeen')}` : ''
+  statusText = subject?.retired ? `${t('Banners.SubjectSetProgressBanner.finished')}` : statusText
+  const progressText = `${t('Banners.SubjectSetProgressBanner.bannerText', { setName })} ${subject?.priority}/${subjectTotal}`
```

Comment:
In RTL languages, like Hebrew, we may need to reverse this entire string, not just the output of the translation function.

Reply:
Good point! I played around with switching locale in storybook and setting the document's body `dir` to 'rtl'. The previous translation setup works better than having to manually reverse the string in SubjectSetProgressBanner. I'll revert to the old string setup.

Reply:
Cool that the storybook lets us test RTL languages. I think Mark has links to at least one live project in Hebrew.